OpenCombine.py

```diff
@@ -0,0 +1,22 @@
+# To use `OpenCombine.py`, figure out its full path.
+# Let's say the full path is `~/projects/OpenCombine/OpenCombine.py`.
+# Then add the following statement to your `~/.lldbinit` file:
+#
+#     command script import ~/projects/OpenCombine/OpenCombine.py
+
+import lldb
+
+# Show a Demand as either `max(N)` or `unlimited`.
+def summary_Demand(sb_value, internal_dict):
```

Comment:
```suggestion
def summary_demand(sb_value, internal_dict):
```

[PEP 8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

Reply:
I prefer keeping the D capitalized here as this method is specifically referring to the Demand type, which is spelled with a capital D.

Reply:
Understood. Okay, it's not like it's crucial here, let's leave it as is then.